File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {
    const stringNumbers = {'zero': 0, 'one': 1, 'two': 2, 'three':3,
                          'four': 4, 'five': 5, 'six': 6, 'seven': 7,
                          'eight': 8, 'nine': 9};
    for ( const [key, value] of Object.entries(stringNumbers) ) {
        s = s.replaceAll(key, value);
    }
    return parseInt(s);
}

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    string_numbers = {
        'zero': '0', 'one': '1', 'two': '2', 'three': '3',
        'four': '4', 'five': '5', 'six': '6', 'seven': '7',
        'eight': '8', 'nine': '9'
    }
    
    for key, value in string_numbers.items():
        s = s.replace(key, value)
                      
    return int(s)